### PR TITLE
[dev-v2.6] rke2: 1.18.20, 1.19.12, 1.20.8, 1.21.2

### PR DESCRIPTION
**Pull request by briandowns**
- rke2: 1.18.20, 1.19.12, 1.20.8, 1.21.2
- go generate

### CODE REVIEW

File: channels-rke2.yaml [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
-  - version: v1.19.11+rke2r1
+  - version: v1.19.12+rke2r1
     minChannelServerVersion: v2.6.0-alpha1
     maxChannelServerVersion: v2.6.99
-  - version: v1.20.7+rke2r2
+  - version: v1.20.8+rke2r1
     minChannelServerVersion: v2.6.0-alpha1
     maxChannelServerVersion: v2.6.99
-  - version: v1.21.1-rc3+rke2r1
+  - version: v1.21.2-rc1+rke2r1
```
Comment (davidnuzik) [inline channels-rke2.yaml]: LGTM after update to remove rc and redo go generate

Comment (davidnuzik) [reply channels-rke2.yaml]: When it's ready for review just include Kinara as a 2nd reviewer. 2.6 is not a priority can do it tomorrow, Thursday the 24th or Friday the 25th.

Comment (briandowns) [reply channels-rke2.yaml]: Sounds good Dave. Thanks. 

Tagging @kinarashah

**Comment by briandowns** [comment]
Ready for review again: @kinarashah @davidnuzik

**Review by davidnuzik — APPROVED** [review]
LGTM

**Review by kinarashah — APPROVED** [review]

**Pull request merged**